akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -307,6 +333,46 @@ final class ORSet[A] private[akka] (
     }
   }
 
+  private def mergeAddDelta(that: ORSet[A]): ORSet[A] = {
```

Comment:
This implementation is not completed at all, I just copied the full merge and made it work for the trivial case. We should probably look at your previous implementation.